src/Sarif/Schemata/sarif-2.1.0-rtm.6.json

```diff
@@ -97,14 +97,16 @@
           "description": "The index within run.addresses of the cached object for this address.",
           "type": "integer",
           "default": -1,
-          "minimum": -1
+          "minimum": -1,
+          "maximum": 2147483647
```

Comment:
>2147483647 [](http://example.com/codeflow?start=21&length=10)

Create a second file to specify .NET

Reply:
fixed to use rtm.5.dotnet
since we are using a version for dotnet specific implementation, I think we don't need to bump the version number to 6 now.